mantidimaging/gui/windows/nexus_load_dialog/presenter.py

```diff
@@ -220,7 +222,10 @@ def _look_for_recon_entries(self):
         assert self.nexus_file is not None
         for key in self.nexus_file.keys():
             if DEFINITION in self.nexus_file[key].keys():
-                if np.array(self.nexus_file[key][DEFINITION]).tobytes().decode("utf-8") == NXTOMOPROC:
+                definition_data = self.nexus_file[key].get(DEFINITION)
+                if definition_data is None:
+                    continue
```

Comment:
I think we could use cast here. We know the `DEFINITION` is there, and I assume from the nexus spec it must be a string. Otherwise I'd rather have an assertion so that we get an error rather than silently continuing.